addons/delivery/data/delivery_data.xml

```diff
@@ -24,5 +24,47 @@
             <field name="delivery_type">fixed</field>
             <field name="product_id" ref="delivery.product_product_delivery"/>
         </record>
+
```

Comment:
You could remove "Normal Delivery Charges" since it's quite a subset of those new deliveries

Reply:
Done, replaced with the "Local Delivery" wherever it showed up.